packages/components/constraints/src/horizontal/horizontal.tsx

```diff
@@ -31,15 +31,15 @@ type TProps = {
    */
   max?: TMaxProp;
   /**
-   * @deprecated: Please use the `max` prop instead. Determines scale of the constraint.
```

Comment:
This can actually be quite useful I think. In the furture :D https://devblogs.microsoft.com/typescript/announcing-typescript-4-0-beta/#deprecated-support

Reply:
I love it
![image](https://user-images.githubusercontent.com/462507/104589677-88687000-566a-11eb-8016-72744120a19b.png)

Reply:
Yup. It's about that. TypeScript basically does a ~value~ in the type's IntelliSense.